tests/braille-specs/ur-pk-g2.yaml

```diff
@@ -59,11 +59,25 @@ tests:
 
 # ARABIC LIGATURE ALLAH ISOLATED FORM (\xFDF2)
   - [ﷲ, ⠁⠠⠇⠈⠓]
-  - [رُوحُﷲ, ⠗⠥⠺⠱⠁⠠⠇⠈⠓]
+  - [رُوحُ ﷲ, ⠗⠥⠺⠱⠁⠠⠇⠈⠓]
+  - [رُوحُﷲ, ⠗⠥⠺⠱⠁⠠⠇⠈⠓, {xfail: "BT"}]
+  - [رُوحُ اﷲ, ⠗⠥⠺⠱⠁⠠⠇⠈⠓, {xfail: "BT"}]
```

Comment:
Are these failure something we intend to fix, or is it difficult to fix? In case of the latter it is better to temporarily change to `testmode: forward` so that the forward part is tested.

Reply:
@bertfrees basically they are spelling variations of the words in line 62 but the forward translation should be the same in each case. So back translate will necessarily fail on these variations. So I wanted to keep them together with the usual/most common spelling (line 62). 
But I have repeated the 2 failing tests in forward mode only in lines 401 and 402.
Does this make sense? Or is there a better way to organise it?
I've done similar in quite a few other places in the tests.

Reply:
@bertfrees perhaps it is better if I move all the "forward only" tests to the places where I have all the "xfails" - temporarily changing the mode as you suggest? I'm happy to do this if it makes it clearer.

Reply:
@bertfrees the big downside is that there will be a lot of chopping and changing. At the moment I have grouped related words together. I will need to add 2 lines before and after each block of what are now "xfails" - it could be even more confusing.

Reply:
> &#x23; All the {xfail:"BT"} repeated for forward only

Ah OK sorry, I missed/forgot about this comment. Sure, it makes sense to do it like this. But what I don't like about this is that the `xfail` are not meant to be ever removed. In this case I think it's better to not use `xfail`. You could change it to a comment:

```yaml
  - [ﷲ, ⠁⠠⠇⠈⠓]
  - [رُوحُ ﷲ, ⠗⠥⠺⠱⠁⠠⠇⠈⠓]
# - [رُوحُﷲ, ⠗⠥⠺⠱⠁⠠⠇⠈⠓] # only forward translation, forward test included below
# - [رُوحُ اﷲ, ⠗⠥⠺⠱⠁⠠⠇⠈⠓] # only forward translation, forward test included below
```

Reply:
Your suggestion sounds good. I will make the changes and commit here in the next day or 2.